RadiationSensorAdapter.java

```diff
@@ -0,0 +1,38 @@
+import java.util.Random;

```

Comment:
Again, this shows that you were not sure what adapter classes mean, or what they are for. Please see this link for a better example:
https://www.youtube.com/watch?v=qG286LQM6BU&t=530s